ui/src/cloud/actions/demodata.ts

```diff
@@ -74,11 +130,19 @@ export const deleteDemoDataBucketMembership = (bucketID: string) => async (
   } = getState()
 
   try {
-    await deleteDemoDataBucketMembershipAJAX(bucketID, userID)
+    await deleteDemoDataBucketMembershipAJAX(bucket.id, userID)
```

Comment:
this seems like a duplicate pattern from `getDemoDataBucketMembership`

Reply:
made single bucket request here as well.